### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by ncdlek**
Initial turkish translation

### CODE REVIEW

File: src/tr.i18n.js [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+    'Copy ASCII Diagram': 'ASCII Diyagramı Kopyala',
+    'Game Info': 'Oyun Bilgisi',
+    'Manage Games…': 'Oyunları Yönet…',
+    'Preferences…': 'Tercihler…'
+  },
+  'updater': {
+    'Download Update': 'Güncellemeyi İndir',
+    'View Changelog': 'Değişiklikleri Görüntüle',
+    'Not Now': 'Şimdi Değil',
+    '${appName} v${version} is available now.': '${appName} v${version} şimdi mevcut',
```
Comment (yishn) [inline src/tr.i18n.js:21]: Interpolated strings use a special syntax. This needs to be 

~~~js
p => `${appName} v${version} şimdi mevcut`
~~~

Please look at the other translation files for more examples.

File: src/tr.i18n.js [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+    'Flatten': 'Düzleştir',
+    'Remove Node': 'Düğümü Kaldır',
+    'Remove Other Variations': 'Diğer Varyasyonları Sil'
+  },
+  'menu.find': {
+    'Find': 'Bul',
+    'Toggle Find Mode': 'Bulma Moduna Geç',
+    'Find Next': 'Sonrakini Bul',
+    'Find Previous': 'Öncekini Bul',
+    'Toggle Hotspot': 'Hotspot ->',
```
Comment (yishn) [inline src/tr.i18n.js:70]: This translation is questionable, can you please make sure this is correct?

File: src/tr.i18n.js (hunk 1 above)
Comment (ncdlek) [reply src/tr.i18n.js]: resolved

File: src/tr.i18n.js (hunk 2 above)
Comment (ncdlek) [reply src/tr.i18n.js]: hotspot and toggle words don't have a direct translation on Turkish. Now I found a way to tanslate.

**Pull request merged**